# Approaches for integrating heterogeneous RNA-seq data reveals cross-talk between microbes and genes in asthmatic patients

Daniel Spakowicz*1,2,3,4, Shaoke Lou*1, Brian Barron1, Jose L Gomez5, Tianxiao Li1, Qing Liu5, Nicole Grant5, Xiting Yan5, Rebecca Hoyd3, George Weinstock2, Geoffrey L Chupp5, Mark Gerstein1,6,7,8

1 Program in Computational Biology and Bioinformatics, Yale University, New Haven, CT
2 The Jackson Laboratory for Genomic Medicine, Farmington, CT
3 Division of Medical Oncology, Ohio State University College of Medicine, Columbus, OH
4 Department of Biomedical Informatics, Ohio State University College of Medicine, Columbus, OH
5 Section of Pulmonary, Critical Care, and Sleep Medicine, Department of Internal Medicine, Yale University School of Medicine, New Haven, CT
6 Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, CT
7 Department of Computer Science, Yale University, New Haven, CT
8 Department of Statistics and Data Science, Yale University, New Haven, CT
* These authors contributed equally

ABSTRACT
Sputum induction is a non-invasive method to evaluate the airway environment, particularly for asthma. RNA sequencing (RNA-seq) can be used on sputum, but it can be challenging to interpret because sputum contains a complex and heterogeneous mixture of human cells and exogenous (microbial) material. In this study, we developed a methodology that integrates dimensionality reduction and statistical modeling to grapple with the heterogeneity. The method, called LDA-link, connects microbes to genes using reduced-dimensionality Latent Dirichlet Allocation (LDA) topics. We validated our method with single-cell RNA-seq and microscopy and then applied it to sputum of asthmatic patients to find known and novel relationships between microbes and genes. We expect this method to be broadly useful for making inferences in heterogeneous and noisy RNA-seq datasets.
